cool.py

```diff
@@ -636,36 +652,23 @@ def update_index(col, row):
     table_id = f"{PROJECT_ID}.indices.images_within_habitat"
 
     #: create dml statement to update specific row
-    dml = f"""
+    sql = sqlalchemy.text(
```

Comment:
question: so our ables are named exactly the same in cloud SQL as they were in BigQuery, it's just that now we're using sqlalchemy to update them instead of the BQ client?

Reply:
yes. except the table didn't contain project and dataset.